### ISSUE TITLE
Navbar item font style not adaptable

### ISSUE THREAD

**Issue by RhoKappa** [benjaminkott/bootstrap_package#1125]
**Prerequisites**
Typo3 10.4.11 with Introduction Package 4.2.0 and Bootstrap Package 11.0.2
I checked the logged issues but did not find a resolution for that specific problem (https://github.com/issues?q=is%3Aissue+user%3Abenjaminkott+repo%3Abenjaminkott%2Fbootstrap_package).

**Description**
In my Typo 3 installation (using the official introduction package), there is no way to customize the navbar item text (nav-link-text) regarding font size or shadow.
The font size of the main navigations on top of the screen is far too small and I intend to add a font shadow, since some alternating backgrounds are light and others are dark.

At first, I tried all kinds of template settings with the constant editor (Bootstrap 4.X: STYLING), e.g. the $navbar-brand-font-size which had, however no effect (of course, I tried flushing all caches, even the PHP cache).
![grafik](https://user-images.githubusercontent.com/78322431/147832049-868e0d0c-7454-4d4c-8102-026dafca40d2.png)

Then I tried by adapting the less file “navbar.less” in path:
…/typo3conf/ext/bootstrap_package/Resources/Public/Less/Theme
Unfortunately with no success.

Afterwards I changed the styles in file “_navbar.scss”
…/typo3conf/ext/bootstrap_package/Resources/Public/Scss/Theme
(for “.nav-link-text” I added “font-size: $font-size-lg; text-shadow: .2em .2em $navbar-dark-color;”
Still, no success.
![grafik](https://user-images.githubusercontent.com/78322431/147832064-fc1ce80e-4bbd-482c-a06d-60a51623c11c.png)

Lastly, I adjusted the settings CSS file “bootstrap4-theme.css” in folder
../typo3conf/ext/bootstrap_package/Resources/Public/Css
Again, no success.
![grafik](https://user-images.githubusercontent.com/78322431/147832078-f53f746f-f372-44e4-96ab-caed70799b1d.png)

I really don’t know if
a)	it is a bug that the constant changes fail to be effective in my installation (and that there is no constant to set shadows for the navbar text), or
b)	my attempts are a completely wrong approach, or
c)	there is just no possibility to change the navbar text style   8-|

I would appreciate any kind of help or tip on where or how to try next  :-)

**Steps to Reproduce**
- show the root page (home) [in preview mode], check main navigation on top
- show any other page [in preview mode], check main navigation on top

**Expected behavior**
On the root page (home) and all other pages, the navbar-items are supposed to be shown in a larger font size and with a text shadow.
**Actual behavior**
On the root page (home) and all other pages, the navbar-items are always shown in the same small font size and with no text shadow, no matter what settings I try.
Screenshots

**Versions**
Typo3 10.4.11
Introduction Package 4.2.0
Bootstrap Package 11.0.2
using Firefox 95.0.2

**Comment by gilbertsoft** [comment]
So far I've understood you do not use a Sitepacke. For your best experience please create a Sitepackage at https://www.sitepackagebuilder.com. Never change the files of the Bootstrap Package, your changes will be lost on the next update, instead make your changes in the Sitepackage.

For your current problem it's enough to adapt some variables in Resources/Public/Scss/Theme/_variables.scss and theme.scss. It's also possible to use the TypoScript configuration for it but you cannot use the constant editor for all the changes you have to make. You can find more information about Sitepackages and how to use it in the docs at https://docs.typo3.org/m/typo3/tutorial-sitepackage/10.4/en-us/.

Hope this helps for your next steps. If not just leave a comment where you stuck.

**Comment by RhoKappa** [comment]
Many thanks for your hint!
For the time being, I defined two new variables `$navbar-nav-link-size` and `$navbar-nav-link-shadow` in file `_variables.scss` of the Bootstrap package (/typo3conf/ext/bootstrap_package/Resources/Public/Scss/Theme) and allocated respective values to it ( I guess, for long term sustainability, I have to create a separate site package).
Then, I added the variables to the `nav-link-text` in file `_navbar.scss` with the following content
![grafik](https://user-images.githubusercontent.com/78322431/147861358-333eed30-8216-40a7-8b59-e43a53ede355.png)

Now the navbar-items are larger and displayed with a shadow :-)
Thanks again!
